js/form.js

```diff
@@ -1,7 +1,14 @@
-$('#show-password').click(function () {
-	if ( $('#password').attr('type') == "password" ) {
-		$('#password').attr('type', 'text');
-	} else {
-		$('#password').attr('type', 'password');
-	}
+// var instead of let/const for better older browsers compatibility
+var passwordTextField;
```

Comment:
Why store $("#password") in a var and not keep it as such all over the place ?

Reply:
I'm not sure what you mean. Something needs to be changed? 
`passwordTextField` is there to avoid scanning DOM at every toggle triggered.

Reply:
No I was just asking why you kept this in a var and you explained it perfectly well above (avoid scanning DOM). Thanks